spec/controllers/state_file/intake_logins_controller_spec.rb

```diff
@@ -421,7 +421,7 @@
 
             expect(subject.current_state_file_az_intake).to eq(intake)
             expect(response).to redirect_to az_questions_data_review_path(us_state: "az")
-            expect(GlobalID.find(session[:state_file_intake])).to eq intake
+            expect(session["warden.user.state_file_az_intake.key"].first.first).to eq intake.id
```

Comment:
i suppose there isn't any guidance for how to access these things in tests? tried a cursory search and couldn't find anything

Reply:
This is lower level than I would like - but it does seem to be the easiest way of validating the session state